### DISCUSSION TITLE
Unable to accept Invite Discord

### DISCUSSION THREAD

**Discussion by Marcus101RR** [Pryaxis/TShock#2493] [General]
Unable to accept Invite Discord. Seems like it doesn't want me to enter the discord, looked at the ban list and I wasn't listed so It isn't that?

**Comment by bartico6** [comment]
Context: Marcus is currently banned.
![obraz](https://user-images.githubusercontent.com/3310937/142503461-02a66bce-c128-40e0-9d0d-6d0e85c10e72.png)

**Comment by Marcus101RR** [comment]
So I take it after all these months still that this is rejected? Since There is no response.

**Comment by hakusaro** [comment]
After performing a detailed and careful analysis, we believe that you have been positively identified to be the culprit behind the original discord server being permanently banned and deleted. Therefore, we will not lift your ban at this time. We deeply apologize for this unsatisfactory result.

**Comment by Marcus101RR** [comment]
You got your own discord server banned, I had nothing to do with it. Wild accusations, I see you haven't changed a bit since 3 years. I expected more out of someone like you.

**Reply by hakusaro** [reply]
> You got your own discord server banned

With all due respect, the information I have is that you are the directly responsible party for this action. It's not that I don't have faith in your information, it's that I trust my sources to not provide false information.

If you would like to be unbanned, please donate a minimum of $50 in USD equivalent of local currency to [GiveWell's Top Charities Fund](https://www.givewell.org/top-charities-fund) and provide a copy of the receipt. I will match your donation up to $500 USD. Then, you will be unbanned on a last-chance basis.

Please let me know if you have any questions about this policy.

**Reply by hakusaro** [reply]
Also, if you'll allow me to make myself clear: Previous iterations of the Discord ToS and Code of Conduct around game cheating, hacks, and ancillary components are diametrically opposed to my beliefs. The fact of the matter is that my account _was_ unbanned, that Discord did undo their action to the extent that they could, and that we have had no problems sense. The only impact is that the server and its history was lost permanently because Discord's philosophy around data backup and recovery is "we don't have that." It is clear that if I was banned for any materially significant breach of the ToS, I wouldn't have had my account fully restored.

**Comment by CharNatorn** [comment]
What did he do to get the server banned?

**Reply by hakusaro** [reply]
That’s really not your concern.
